pouchdb-core/pouchdb-core.d.ts

```diff
@@ -13,9 +16,9 @@ declare namespace PouchDB {
         type AnyCallback = Callback<any, any>;
         type DocumentId = string;
         type DocumentKey = string;
+        type AttachmentId = string;
         type RevisionId = string;
         type Availability = 'available' | 'compacted' | 'not compacted' | 'missing';
-        type Attachment = string | ArrayBuffer;
```

Comment:
This type is intended to reflect the base64/Blob/Buffer attachments that can be used, [as described in this part of the docs](https://pouchdb.com/api.html#save-a-base64-attachment).